.github/workflows/ci.yml

```diff
@@ -40,6 +40,7 @@ jobs:
           - "tag_array_64x184_report"
           - "lb_32x128_generate_abstract"
           - "lb_32x128_test_generate_abstract"
+          - "lb_32x128_generate_abstract_mock_area"
```

Comment:
We could make this only an action and not a new target, though debugging locally with _mock_area_deps could be useful...

Reply:
Do you mean to not test `lb_32x128_generate_abstract_mock_area` target as separate job in matrix element? Or to not create `*_mock_area` targets at all?

Reply:
not to create *_mock_area at all. To the user this is an implementation detail.

Reply:
This target is no longer present